apps/probate/probate-cron-make-dormant-cases/probate-cron-make-dormant-cases.yaml

```diff
@@ -9,7 +9,7 @@ spec:
       environment:
         TASK_NAME: MakeDormantCasesTask
       useInterpodAntiAffinity: true
-      image: hmctspublic.azurecr.io/probate/back-office:prod-3404fc4-20221130141802 #{"$imagepolicy": "flux-system:probate-back-office"}
+      image: hmctspublic.azurecr.io/probate/back-office:prod-3404fc4-20221130141802 #{"$imagepolicy": "flux-system:demo-probate-back-office"}
```

Comment:
Isn't this going to make prod point to demo too?

Reply:
Prod is not still there .I will revert once testing is done